src/Kmd.Logic.Gateway.Automation/Models/ApiValidationModel.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.IO;
+
+namespace Kmd.Logic.Gateway.Automation.Models
+{
+    public class ApiValidationModel
+    {
+        public ApiValidationModel(
```

Comment:
I think we should include ApiverssionetId also as part of this , so that when we support adding versions that would help

Reply:
But in publish.yaml file there is no ApiVersionSetId property. We have Version and IMO it's enough. Give me case where ApiVersionSetId would be helpful instead of just Version.

Reply:
Agreed publish.yaml file doesn't have ApiVersionSetId , the post Api of api creation takes ApiVersionSetId as  nullable query parameter, it works in such a way that if ApiVersionSetId is passed null it creates new api and when ApiVersionSetId is passed it creates new version for the api. So we need to rethink about how our yaml looks like.  If this is considered then no problem.

Reply:
Ok, then you are right that we should add ApiVersionSetId to response model.

Reply:
done